front-end/components/views/Auth/SignUpView.jsx

```diff
@@ -55,15 +58,19 @@ class SignUpView extends React.Component {
             onChange={this.setUsername}>
           </TextEntry>
         </div>
+
         <div style={{ height: '5vw' }}></div>
```

Comment:
Do not use divs as Spacer() from swift UI.  Instead modify the margin on the element you want to give more space to.  In this case you could set the margin of the className "login-field" to a minimum pixel height.  The more divs used the slower the website load time. Also idk if you did this on purpose, but you set the height parameter dependent on the width of the screen '5vw' means 5% of the view width.